tests/integration/test_role/test.py

```diff
@@ -418,72 +419,215 @@ def test_function_current_roles():
     )
 
 
-def test_role_expiration():
-    instance.query("CREATE USER ure")
+@pytest.mark.parametrize("with_extra_role", [False, True])
+def test_role_expiration(with_extra_role):
```

Comment:
Here I combined tests `test_role_expiration` and `test_two_roles_expiration` together.